include/beman/any_view/detail/unique_address.hpp

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef BEMAN_ANY_VIEW_DETAIL_UNIQUE_ADDRESS_HPP
+#define BEMAN_ANY_VIEW_DETAIL_UNIQUE_ADDRESS_HPP
+
+//clang-format off
+#if !defined(__has_include) || __has_include(<beman/any_view/config.hpp>)
```

Comment:
If `__has_include` isn't defined, wouldn't this be syntactically ill-formed? I thought it was necessary to nest these, i.e.
```suggestion
#ifdef __has_include
#if __has_include(<beman/any_view/config.hpp>)
...
```